bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/GetLedgerMetaService.java

```diff
@@ -45,12 +48,12 @@
     static final Logger LOG = LoggerFactory.getLogger(GetLedgerMetaService.class);
 
     protected ServerConfiguration conf;
-    protected ZooKeeper zk;
-
-    public GetLedgerMetaService(ServerConfiguration conf, ZooKeeper zk) {
+    protected LayoutManager layoutManager;
+    // TODO: remove zk?
+    public GetLedgerMetaService(ServerConfiguration conf, LayoutManager layoutManager) {
```

Comment:
This service should just take LedgerManagerFactory from Bookie

Reply:
Thanks, will change it.